pages/finance/creating_purchase_requests.mdx

```diff
@@ -0,0 +1,23 @@
+# Creating Purchase Requests
+
+1. Go to https://finance-frontend.watonomous.ca/
+2. Create New Ticket > Ticket Type: UW Finance Purchase
+3. Enter all purchase details. For funding item link, you would either be using FI-1 (WATO Cash) or one of the Funding Items that match the category. For example, if you're purchasing an RTX 4090 it would probably fit under GPU funding.
+4. Await team approval. Please reach out to your team director, team captain, and the faculty advisor.
+5. Once this has been completed, the status will transition to `READY_FOR_BUY`. You should purchase the item at this step.
+6. Once the item has been purchased, please upload the receipt, as well as a University of Waterloo Finance reimbursement form. Instructions found below.
+7. Click Confirm Item(s) Purchased and Submit Reimbursement
+
+## Creating a University of Waterloo Finance reimbursement form.
+
+1. Navigate [here](https://uwaterloo.ca/engineering-endowment-foundation/getting-funding/spending-funding/submitting-reimbursement-request)
+2. Click the link in step 1 titled "reimbursement form (excel)"
+3. Enter the following fields
+4. Payee is Student
+5. Claimant Name, Student Number, Department, Phone Number, E-mail Address
+6. Mailing Address, City, Province/State, Postal Code. For these, use the address your items were shipped to, or your personal address.
+7. Destination/Reason for Request: Please put the corresponding reference number. Reach out to @smileycow on Discord if you do not know what this is
```

Comment:
for my knowledge, is the reference number the PPR code or is it something else?

Reply:
this is the reference code from weef